<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>275</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>641</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>选择内容</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>简单的ComboBox</string>
   </property>
   <widget class="QPushButton" name="pushButton_InitList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearList">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>清除列表</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>105</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>可编辑</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_City">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>绵阳市</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>达州市</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>德阳市</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>资阳市</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>成都市</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>带有用户数据的ComboBox</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_InitCity">
       <property name="text">
        <string>初始化城市</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_User"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/images/132.bmp</normaloff>:/new/prefix1/images/132.bmp</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
